Patches/CheckGameEndPatch.cs

```diff
@@ -576,6 +583,11 @@ public bool CheckGameEndByLivingPlayers(out GameOverReason reason)
                     reason = GameOverReason.HumansByVote;
                     winner = CustomWinner.Crewmate;
                 }
+                else if (Jackal < Si && Pel < Si && Imp < Si && BK < Si && Gam < Si && Crew < Si && CM < Si) //胜利
+                {
+                    reason = GameOverReason.ImpostorByKill;
+                    CustomWinnerHolder.ResetAndSetWinner(CustomWinner.Yandere);
```

Comment:
It seems the victory condition was not done correctly, look at what the victory condition looks like for other NKs and do the same

Reply:
That should do it.